<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuWindow</class>
 <widget class="QMainWindow" name="MenuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MenuWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-20</y>
      <width>1151</width>
      <height>791</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="background">
      <property name="geometry">
       <rect>
        <x>-80</x>
        <y>-20</y>
        <width>1300</width>
        <height>750</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/img/img/bg1.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="playBtn">
      <property name="geometry">
       <rect>
        <x>494</x>
        <y>623</y>
        <width>129</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
border-radius: 1px;
color: black;
text-transform: uppercase;
</string>
      </property>
      <property name="text">
       <string>Start Game</string>
      </property>
     </widget>
     <widget class="QLabel" name="main_title">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>35</y>
        <width>461</width>
        <height>161</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/img/img/logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="pokedex">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>170</y>
        <width>941</width>
        <height>571</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/img/img/pokedex.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>320</y>
        <width>241</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="spacing">
        <number>3</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="font">
            <font>
             <pointsize>7</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #D2242A;
color: white;
padding: 5px;
text-transform: uppercase;</string>
           </property>
           <property name="text">
            <string>Pokemon Library File</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="pkmnLibEdit">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: #eeeeee;
color: black;
padding: 5px;</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pkmnLibBtn">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">background: #eeeeee;
padding: 5px;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/img/img/Pokeball.png</normaloff>:/img/img/Pokeball.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="font">
            <font>
             <pointsize>7</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #D2242A;
color: white;
padding: 5px;
text-transform: uppercase;</string>
           </property>
           <property name="text">
            <string>Move Library File</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="moveLibEdit">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: #eeeeee;
color: black;
padding: 5px;</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="moveLibBtn">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">background: #eeeeee;
padding: 5px;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/img/img/Pokeball.png</normaloff>:/img/img/Pokeball.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_15">
           <property name="font">
            <font>
             <pointsize>7</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #D2242A;
color: white;
padding: 5px;
text-transform: uppercase;</string>
           </property>
           <property name="text">
            <string>Game Command File</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="gameDataEdit">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: #eeeeee;
color: black;
padding: 5px;</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="gameDataBtn">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">background: #eeeeee;
padding: 5px;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/img/img/Pokeball.png</normaloff>:/img/img/Pokeball.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="caseBtn">
      <property name="geometry">
       <rect>
        <x>428</x>
        <y>218</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent; 
border: 0px;
text-transform: uppercase;
color: #eeeeee;</string>
      </property>
      <property name="text">
       <string>Case</string>
      </property>
     </widget>
     <widget class="QLabel" name="caseLabel">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>535</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>151</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
color: #D2242A;
padding: 5px;
text-transform: uppercase;</string>
      </property>
      <property name="text">
       <string>Manual Input</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="resetBtn">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>579</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>6</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent; 
border: 0px;
text-transform: uppercase;
color: #eeeeee;</string>
      </property>
      <property name="text">
       <string>RESET</string>
      </property>
     </widget>
     <widget class="QLabel" name="oak">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>230</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/img/img/oak.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="main_title_3">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>200</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/sprites/sprites/pikachu-original-cap.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="enemy">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>290</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/img/img/trainer_0.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="pkmn_enemy">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>261</width>
        <height>251</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/sprites/sprites/eevee.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="leftBtn">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>613</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent; 
border: 0px;
text-transform: uppercase;
color: #eeeeee;</string>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rightBtn">
      <property name="geometry">
       <rect>
        <x>707</x>
        <y>611</y>
        <width>29</width>
        <height>26</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent; 
border: 0px;
text-transform: uppercase;
color: #eeeeee;</string>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="enemyLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>520</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
background: rgba(0, 0, 0, 50);
text-transform: uppercase;</string>
      </property>
      <property name="text">
       <string>ENEMY : GARY</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="player">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>300</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/img/img/trainer_you.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="pikachu">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>280</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/sprites/sprites/pikachu.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="testBtn">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>250</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent; 
border: 0px;
text-transform: uppercase;
color: #eeeeee;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="testLabel">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>210</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
background: rgba(0, 0, 0, 0);
padding: 5px;
text-transform: uppercase;</string>
      </property>
      <property name="text">
       <string>TEST MODE OFF</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="pkmn_enemy_3">
      <property name="geometry">
       <rect>
        <x>709</x>
        <y>269</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/sprites/sprites/squirtle.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="pkmn_enemy_4">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>340</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/sprites/sprites/zorua-hisui.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="pkmn_enemy_5">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>360</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/sprites/sprites/bulbasaur.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="pkmn_enemy_6">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>130</y>
        <width>241</width>
        <height>261</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/sprites/sprites/charizard.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="pkmn_enemy_7">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>230</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/sprites/sprites/pidgeot.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="pkmn_enemy_8">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>20</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/sprites/sprites/butterfree.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="pkmn_enemy_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/sprites/sprites/mew.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="pkmn_enemy_10">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>-70</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/sprites/sprites/celebi.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="pkmn_enemy_11">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>-70</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/sprites/sprites/jirachi.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="enemyRemind">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>660</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>6</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
background: rgba(0, 0, 0, 0);
padding: 5px;
text-transform: uppercase;</string>
      </property>
      <property name="text">
       <string>SELECT ENEMY</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <zorder>background</zorder>
     <zorder>pkmn_enemy_3</zorder>
     <zorder>main_title</zorder>
     <zorder>pokedex</zorder>
     <zorder>playBtn</zorder>
     <zorder>verticalLayoutWidget_2</zorder>
     <zorder>caseBtn</zorder>
     <zorder>caseLabel</zorder>
     <zorder>resetBtn</zorder>
     <zorder>oak</zorder>
     <zorder>main_title_3</zorder>
     <zorder>enemy</zorder>
     <zorder>pkmn_enemy</zorder>
     <zorder>leftBtn</zorder>
     <zorder>rightBtn</zorder>
     <zorder>enemyLabel</zorder>
     <zorder>testLabel</zorder>
     <zorder>testBtn</zorder>
     <zorder>pkmn_enemy_4</zorder>
     <zorder>pkmn_enemy_6</zorder>
     <zorder>pkmn_enemy_7</zorder>
     <zorder>pkmn_enemy_9</zorder>
     <zorder>pkmn_enemy_10</zorder>
     <zorder>pkmn_enemy_11</zorder>
     <zorder>player</zorder>
     <zorder>pkmn_enemy_5</zorder>
     <zorder>pikachu</zorder>
     <zorder>pkmn_enemy_8</zorder>
     <zorder>enemyRemind</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
